config.py

```diff
@@ -60,6 +60,13 @@ def init_app(app):
         template_folders = [
             os.path.join(repo_root, 'app', 'templates'),
             os.path.join(repo_root, 'node_modules', 'digitalmarketplace-govuk-frontend', 'govuk-frontend'),
+            os.path.join(
+                repo_root,
+                'node_modules',
+                'digitalmarketplace-govuk-frontend',
+                'digitalmarketplace',
```

Comment:
it is coming from digitalmarketplace-govuk-frontend v0.2.0. 

Ideally, we should just do the following and use the folders to namespace our components.

Eg.
`os.path.join(repo_root, 'node_modules', 'digitalmarketplace-govuk-frontend'`

```
{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
{% include "digitalmarketplace/templates/layouts/_custom_dimensions.html" %}
```

I think for the time being we should leave it as it is because v3 of govuk-frontend will require us to revisit this issue and update it to a single entry-point.

Reply:
I think it might need an explanatory comment. We obviously have a lot of things called `digitalmarketplace`, it might not be immediately obvious which one this refers to 😹 

Perhaps pulling out the list of template names into a constant would be useful?

```
FRONTEND_TEMPLATE_FOLDERS = [
    'node_modules', 
    'digitalmarketplace',   # Explain what this is here
    'digitalmarketplace-govuk-frontend',
    'templates'
]

os.path.join(repo_root, **FRONTEND_TEMPLATE_FOLDERS)
```

Reply:
Just writing all that out gave me a better understanding of what's going on 😹